prime-router/metadata/fhir_mapping/hl7/datatype/Entity_Identifier_Pair_2.yml

```diff
@@ -5,7 +5,7 @@ type:
    expressionType: resource
    specs: $type
    vars:
-     valueIn: $code | $alternateCode | $type
+     valueIn: $code | $primaryCode | $type
```

Comment:
Was this part of the change intentional?

Reply:
Ah no, I think that needs to get put back

Reply:
Actually I was wrong here, this change is correct as we want to set the first coding in teh codeable concept

Reply:
I think I understand, basically the mapping for SPM.2 goes to two distinct identifiers, neither of which is exactly an "alternate-identifier", so we can/should call them both primaries? Looks like `code` in practice is never actually used here, maybe that's where some of my confusion comes from (`code` and `primaryCode` both sound like primary codes). Overall, the SPM.2 mapping seems out-of-sync with the spec and a bit confusing (maybe this shouldn't map to CodeableConcept at all), but it works for HL7->FHIR-HL7, and it's out of scope for this ticket, so I all good for now.